### ISSUE TITLE
Add a program to sort an integer array in Python

### ISSUE THREAD

**Issue by AkshayCHD** [pclubuiet/github101#111]
Input an integer array and sort its elements.

**Comment by vineet8588** [comment]
I dont think there are arrays in python by default, there are lists, do we have to use lists or use numpy module?

**Comment by AkshayCHD** [comment]
Yes you may use a list but try to sort it without built in sort function
